centos-7-minimal.ks

```diff
@@ -65,10 +66,29 @@ shim
 LANG="en_US"
 echo "%_install_lang $LANG" > /etc/rpm/macros.image-language-conf
 
+# Make sure to properly handle the ssh keys passed to us via docker-machine
+cat > /etc/rc.d/init.d/copy-ssh << EOF
+
+# Create a docker user
+useradd -p docker docker
+
+# Read /userdata.tar with ssh keys
+dd if=/dev/sda of=/userdata.tar bs=1 count=4096 2>/dev/null
+tar xf /userdata.tar -C /home/docker/ > /var/log/userdata.log 2>&1
+rm -f '/home/docker/boot2docker, please format-me'
+#chown -R docker:staff /home/docker
+EOF
+
+chmod 755 /etc/rc.d/init.d/copy-ssh
```

Comment:
@hferentschik I tested a simple script to run at boot and it seems to work fine. Here are the steps needs to be done 
```
chmod +x /etc/rc.d/rc.local
make the script executable i.e. chmod +x <script>
make an entry in /etc/rc.local
```
Post POC we should definitely use systemd as the backward compatibility mode makes it difficult for debug issues. Creating systemd service files should not very difficult too.